ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -294,6 +294,10 @@ def greater_than(self, y, name=None):
     def rsqrt(self, name=None):
         return ivy.reciprocal(ivy.sqrt(self._ivy_array))
 
+    @with_unsupported_dtypes({"2.5.0 and below": ("float16", "bfloat16")}, "paddle")
+    def divide(x, y, name=None):
+        return paddle_frontend.divide(x, y)

```

Comment:
A frontend function should be made up of ivy functions. So this should be something like 'ivy.divide'

Reply:
hey @Sam-Armstrong  i made changes to the dtype to float32 and float64. now it's working all test are passing.